src/gsModeling/gsBarrierPatch.h

```diff
@@ -0,0 +1,96 @@
+/** @file gsBarrierPatch.h
+
+    @brief This software facilitates the creation of analysis-suitable
+    parameterizations from given boundary representations. Serving as a
+    reference implementation, it embodies the methods and concepts detailed
+    in Ye Ji's doctoral research.
+
+    This file is part of the G+Smo library.
+
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+    Author(s): Ye Ji, H.M. Verhelst
+*/
+
+#pragma once
+
+#include <gismo.h>
+#include <gsHLBFGS/gsHLBFGS.h>
```

Comment:
Maybe you need
```
#ifdef gsHLBFGS_ENABLEDMaybe you need
```
#ifdef gsHLBFGS_ENABLED
#include <gsHLBFGS/gsHLBFGS.h>
#endif
```
but probably the include can be completely removed.
#include <gsHLBFGS/gsHLBFGS.h>
#endif
```
but probably the include can be completely removed.